server/internal/adapter/gql/loader_item.go

```diff
@@ -32,7 +33,7 @@ func (c *ItemLoader) Fetch(ctx context.Context, ids []gqlmodel.ID) ([]*gqlmodel.
 	}
 
 	return lo.Map(res, func(m item.Versioned, i int) *gqlmodel.Item {
-		return gqlmodel.ToItem(m.Value())
+		return gqlmodel.ToItem(m.Value(), false)
```

Comment:
```suggestion
		return gqlmodel.ToItem(m.Value(), sf.Multiple())
```

1. Get `[]id.SchemaID` from the result of `c.usecase.FindByIDs` (item.List)
2. Load schemas from the ID slice.
3. Set the schema field is `multiple` or not to the second argument of `ToItem` func with the corresponding schema field.